README.md

```diff
@@ -330,49 +330,6 @@ and alpha release can also be cut using:
 yarn release-alpha
 ```
 
-### Acceptance testing
```

Comment:
IIRC @sneridagh prefers to have everything needed to get started in the README.md. At the same time, we want to push the new Plone 6 docs instead of maintaining content in multiple places. Would it be appropriate to retain this heading with a link to https://6.dev-docs.plone.org/volto/developer-guidelines/acceptance-tests.html?